docs/resource_sources.rst

```diff
@@ -23,6 +23,9 @@ To see all available public sources for gnomAD resources, use:
 
     list(GnomadPublicResourceSource)
 
+.. note::
+
+   The gnomAD project's bucket (``gs://gnomad-public-requester-pays``) is `requester pays <https://cloud.google.com/storage/docs/requester-pays>`_, meaning that charges for data access and transfer will be billed to your Google Cloud project.
```

Comment:
Should we add in something like 
```suggestion
   The gnomAD project's bucket (``gs://gnomad-public-requester-pays``) is `requester pays <https://cloud.google.com/storage/docs/requester-pays>`_, meaning that charges for data access and transfer will be billed to your Google Cloud project. Clusters must be configured to read requester pays buckets during creation.
``hailctl dataproc start cluster-name --packages gnomad --requester-pays-allow-buckets gnomad-public-requester-pays``
```
since we are removing the example above.